packages/tooling/fast-tagged-templates/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "fast-tagged-templates",
```

Comment:
I like the name, I think this was the primary outstanding question I had. 

I will say that for the package name, I don't think these have to match in this case. I'd prefer an additional layer to note this as either an extension or plugin. Without that, this could mean quite a bit in the tooling repo. While I think in most all cases it makes sense for our packages to share a folder name and `package.json` name, there are reasons for deltas. One similar delta is the eslint plugin. Eslint looks for a specific pattern of name so we have a mismatch there. Similarly, since this is a vscode extension and vscode uses this for the *name* of the extension, I don't know that we have to have the package name be the same as the name in the `package.json`. 

TLDR;

Can we rename the package to be something a bit more verbose and specific to this being an extension? I think clarity will help in the context of the repo. I think the name here is great for the official name of the extension itself and would be clear in the context of vscode as an extension.

Reply:
@radium-v If we can get the name updated here I think this is ready to publish beyond just a draft.

Reply:
We've got a meeting coming up to discuss some productivity features for a FAST VS Code plugin. I'm wondering if we should wrap all of this up in the same plugin. In other words, should we rename this to basically be "the" FAST plugin, and then just place all features together so customers have a single plugin to install for the full experience?